apps/site/src/_pages/docs/0_index.mdx

```diff
@@ -8,7 +8,7 @@ The following pages provide a practical introduction for those:
 
 - [using blocks](https://blockprotocol.org/docs/using-blocks) within applications that support the Block Protocol
 - [building blocks](https://blockprotocol.org/docs/developing-blocks) and publishing them to the [Þ Hub](https://blockprotocol.org/hub)
-- [building block-based applications](https://blockprotocol.org/docs/embedding-blocks) that leverage the Block Protocol
+- [building block-based applications](https://blockprotocol.org/docs/using-blocks) that leverage the Block Protocol
```

Comment:
Can we #anchor-link to the specific location on the page?

Reply:
Yes – https://github.com/blockprotocol/blockprotocol/commit/a99d4f2457ac5c73af8b6b4d164455591eaf0b0a